apps/daisy/settings.js

```diff
@@ -3,40 +3,40 @@
 
   // initialize with default settings...
   let s = {'gy' : '#020',
-           'fg' : '#0f0',
-           'color': 'Green',
-           'check_idle' : true,
-           'batt_hours' : false,
-           'hr_12' : false,
-           'ring' : 'Steps',
-           'idxInfo' : 0,
-           'step_target' : 10000};
+    'fg' : '#0f0',
+    'color': 'Green',
+    'check_idle' : true,
+    'batt_hours' : false,
+    'hourly_buzz' : false,
+    'ring' : 'Steps',
+    'idxInfo' : 0,
+    'step_target' : 10000};
 
-  // ...and overwrite them with any saved values
-  // This way saved values are preserved if a new version adds more settings
-  const storage = require('Storage');
-  let settings = storage.readJSON(SETTINGS_FILE, 1) || s;
-  const saved = settings || {};
-  for (const key in saved) {
-    s[key] = saved[key];
-  }
+// ...and overwrite them with any saved values
+// This way saved values are preserved if a new version adds more settings
+const storage = require('Storage');
+let settings = storage.readJSON(SETTINGS_FILE, 1) || s;
+const saved = settings || {};
+for (const key in saved) {
+s[key] = saved[key];
+}
 
-  function save() {
-    settings = s;
-    storage.write(SETTINGS_FILE, settings);
-  }
+function save() {
+settings = s;
+storage.write(SETTINGS_FILE, settings);
```

Comment:
The indent here and in the above `for` loop seems to have been lost in a reformat:
```suggestion
  settings = s;
  storage.write(SETTINGS_FILE, settings);
```

Reply:
Cleaned up